src/infrastructure/jira/jira-client/jira-client.ts

```diff
@@ -109,15 +110,17 @@ class JiraClient {
 		connectInstallation: ConnectInstallation,
 	): Promise<GetIssuePropertyResponse> => {
 		const url = new URL(
-			`/rest/api/2/issue/${issueIdOrKey}/properties/${propertyKey}`,
+			`${ISSUE_PROPERTY_BASE_URI}/${issueIdOrKey}/properties/${propertyKey}`,
 			connectInstallation.baseUrl,
 		);
 		let response: AxiosResponse<GetIssuePropertyResponse>;
 		try {
 			response = await axios.get<GetIssuePropertyResponse>(url.toString(), {
-				headers: new AxiosHeaders().setAuthorization(
-					this.buildAuthorizationHeader(url, 'GET', connectInstallation),
-				),
+				headers: new AxiosHeaders()
+					.setAuthorization(
+						this.buildAuthorizationHeader(url, 'GET', connectInstallation),
+					)
+					.setAccept('application/json'),
```

Comment:
According to this [jira doc](https://developer.atlassian.com/cloud/jira/platform/jira-entity-properties/) entity properties must be of valid JSON format (anything that JSON.parse can read). 
[Get issue property](https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issue-properties/#api-rest-api-2-issue-issueidorkey-properties-propertykey-get) example also lists in the example to explicitly set the `Accept` header to `application/json`. So I have done this. Before, if this header wasn't set and a `string` was provided here, it would encode as text not json